app/assets/stylesheets/styles/lines.scss

```diff
@@ -12,10 +12,10 @@
   }
 
   label {
-    font-size: $font-size-medium;
+    font-size: variables.$font-size-medium;
   }
 
   .form-check-label {
-    @extend .ml-8;
+    @extend .ml-5;
```

Comment:
for some reason spacers are being applied differently (properly?) now; stuff over 5 gets a little spotty or doesn't acknowledge/style correctly, and other stuff seems to be a little more aggressive/spacious. (There are a bunch of corresponding HTML changes to reflect this, but it all more or less shakes out in the end.)